tests/core/schema/postgresql-permission-1.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE IF NOT EXISTS user_principals (
+    user_id TEXT,
```

Comment:
I'm confused. Why isn't this `COLLATE "C"` too?

Reply:
This is the original schema, from which I want to verify that migration is possible.